tests/test_runners/test_nunit.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+import responses  # type: ignore
+import json
+import gzip
+from tests.cli_test_case import CliTestCase
+
+
+class NUnitTest(CliTestCase):
```

Comment:
I'm not familiar with NUnit. So could you add comments how to output these XML files or add README.md?